recipes/cityhash/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.0.1":
+    url: "https://github.com/google/cityhash/archive/8eded14d8e7cabfcdb10d4be35d521683edc0407.zip"
+    sha256: "e49f5bdb0f93d303bf18cb72f2f18e10ac3b4f2734da7cc1d708f7f2d2674c7c"
```

Comment:
Could you move this version before cci.20130801 please, since 1.0.1 is older than cci.20130801.